Neos.Flow/Classes/ResourceManagement/ResourceManager.php

```diff
@@ -171,8 +187,7 @@ public function importResource($source, $collectionName = ResourceManager::DEFAU
             throw new Exception(sprintf('Importing a file into the resource collection "%s" failed: %s', $collectionName, $exception->getMessage()), 1375197120, $exception);
         }
 
-        $this->resourceRepository->add($resource);
-        $this->logger->debug(sprintf('Successfully imported file "%s" into the resource collection "%s" (storage: %s, a %s. SHA1: %s)', $source, $collectionName, $collection->getStorage()->getName(), get_class($collection), $resource->getSha1()));
+        $this->persistResource($resource, $collection, is_resource($source) ? get_resource_type($source) : sprintf('file "%s"', $source));
```

Comment:
Not sure it's okay to change this slightly, but before importing a PHP `resource` would log something like `Successfully imported file "Resource id #5" into the resource collection ...`. With this, it will become `Successfully imported stream into the resource collection ...`. We can change it back though.

Reply:
Fine with me…